src/Account/Account.php

```diff
@@ -5,25 +5,18 @@
 /**
  * @author Milan Matějček
  */
-class Fio
+class Account
 {
+	/** @var Bank */
+	private $account;
 
 	/** @var string */
 	private $token;
 
-	/** @var Bank */
-	private $account;
-
-	public function __construct($token, Bank $account)
+	public function __construct($account, $token = NULL)
```

Comment:
I know that you use it for validation, but I think there is no need to create instance just for getting few details. I can make `$token` mandatory, then you will merge it?